lib/utils/request.js

```diff
@@ -31,30 +31,42 @@ var async = require('async');
 
 /**
  * Function To Generate A Relevant Error Message
- * @param statusCode - HTTP Code For The Error Message
+ * @param err          - Error from requestJS
+ * @param httpResponse - HTTP Response
  * @param body - Error Response Body
  */
-function createErrorMessage(statusCode, body){
+function createErrorMessage(err, httpResponse, body){
+  var requestId = httpResponse ? httpResponse.headers['x-fh-request-id'] : null;
```

Comment:
Are header keys all lowercased in this object? I think the default was uppercase in fh-logger other than for connect (as opposed to express) modules.